blocks/cards/cards.css

```diff
@@ -1,31 +1,177 @@
+:root {
+  --mt-cards-grid-column-gap: 30px;
+  --mt-cards-grid-row-gap: 35px;
+
+  /* font sizes */
+  --mt-cards-title-font-size: 21px;
+  --mt-cards-text-font-size: 14px;
+
+  /* font family */
+  --mt-cards-font-family: (--mt-body-font-family);
+
+  /* font color */
+  --mt-cards-title-font-color: var(--mt-global-color-base-primary);
+
+  /* colors */
+  --mt-cards-box-shadow-color: #1c1c4940;
+}
+
 .cards > ul {
   list-style: none;
-  margin: 0;
-  padding: 0;
+  width: 100%;
+  align-content: flex-start;
+  box-sizing: border-box;
+  padding: 10px;
+  margin: 3% 0 0;
   display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
-  grid-gap: 16px;
+  gap: var(--mt-cards-grid-row-gap) var(--mt-cards-grid-column-gap);
+  grid-template-columns: repeat(1,1fr);
 }
 
-.cards > ul > li {
-  border: 1px solid var(--mt-global-color-base-secondary);
-  background-color: var(--mt-background-color-primary)
+
+.cards.block > ul > li {
+  padding: 5%!important;
+  box-shadow: 2px 12px 28px 0 var(--mt-cards-box-shadow-color);
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  transition-property: background,border,box-shadow;
+  transition-duration: .25s;
+  align-items: center;
 }
 
-.cards .cards-card-body {
-  margin: 16px;
+.cards > ul > li img {
+  width: 100%;
+  aspect-ratio: 4 / 3;
+  object-fit: cover;
 }
 
-.cards .cards-card-image {
-  line-height: 0;
+.cards.wideimage.block > ul > li {
+  padding: 0!important;
+}  
+
+.cards.block > ul > li:hover {
+  box-shadow: 2px 4px 12px 0 var(--mt-cards-box-shadow-color);
 }
 
-.cards .cards-card-body > *:first-child {
-  margin-top: 0;
+.section.cards-container {
+  padding: 0% 5% 10%;
 }
 
-.cards > ul > li img {
+.section.cards-container .cards-wrapper {
+  max-width: 767px;
+  display: flex;
+  margin-right: auto;
+  margin-left: auto;
+  position: relative;
+  flex-wrap: wrap;
+}
+
+.cards-wrapper > .cards.block > ul > li {
+  position: relative;
+  cursor: pointer;
+}
+
+.cards-card-image {
+  margin-bottom: 20px;
+  height: auto!important;
```

Comment:
please don't use `!important`, for details see this article: https://uxengineer.com/css-specificity-avoid-important-css/